README.md

```diff
@@ -1,43 +1,269 @@
-# pulumi-converter-k8s
+# pulumi-converter-kubernetes
 
-A Pulumi converter plugin to convert kubernetes manifests to Pulumi languages. The idea for this converter is to replace [kube2pulumi](https://github.com/pulumi/kube2pulumi). Currently work in progress.
+The Pulumi Converter Kubernetes plugin facilitates the conversion of Kubernetes manifests into Pulumi languages, enhancing your Kubernetes development workflow. By leveraging strong types, compilation errors, and comprehensive IDE support, such as autocomplete, you can effectively declare and manage infrastructure across various clouds within the same program that handles your Kubernetes resources.
+
+This converter supersedes [kube2pulumi](https://github.com/pulumi/kube2pulumi).
 
 ### Installation
-Install the plugin from Github releases using the following command
-```
-pulumi plugin install converter k8s
+To utilize the pulumi-converter-kubernetes plugin for converting Kubernetes YAML to Pulumi and subsequently deploying it, you must first install the Pulumi CLI.
```

Comment:
We could phase this more simply. EG:

#### Prerequisites
- [Download & Install Pulumi | Pulumi Docs](https://www.pulumi.com/docs/install/)